### ISSUE TITLE
Google Tranlation

### ISSUE THREAD

**Issue by DanielSon33** [lukePeavey/SplitType#67]
First of all, thanks for the great script. It works great on my site. I only have one major issue. When a person uses the google translator, all the text that is broken down into words and chars will not translate. Actually even worse:

Original:
<img width="760" alt="image" src="https://github.com/lukePeavey/SplitType/assets/148707616/35131a99-cc1f-4bfb-88fb-4ae97f657e9c">

After using google translator:
<img width="785" alt="image" src="https://github.com/lukePeavey/SplitType/assets/148707616/4aa11ead-067f-43d2-b820-75da82d0fcbb">

Any idea how to fix this?

Cheers, Daniel

**Comment by lukePeavey** [comment]
Yeah, the Google Translate browser extension **will not work** on split text.
